### PR TITLE
endpoint for assigning members to hedvig forever

### PR DESCRIPTION
**Pull request by FredrikAreschoug**
1. Enabling hedvig forever :partying_face: 
2. Send notification :rocket: 
3. :couch_and_lamp: 

[Notion ticket](https://www.notion.so/hedviginsurance/71cf10addf9e42a2ae4e3e11d1d47a5d?v=698b1693c73d488996e38a2f36af5e58&p=99bfeb9f152c4810ae3cb0067db50c3e)

### CODE REVIEW

**Comment by FredrikAreschoug** [comment]
Don't know what happened with travis-github connection but they are green when look in travis :confused:

File: src/main/java/com/hedvig/productPricing/campaign/service/RedeemCampaignService.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.hedvig.productPricing.campaign.service
+
+import com.hedvig.contract.shared.models.objects.Market
+
+interface RedeemCampaignService {
```
Comment (meletios) [inline src/main/java/com/hedvig/productPricing/campaign/service/RedeemCampaignService.kt:5]:
```suggestion
interface EnablingHedvigForeverService {
```

File: src/main/java/com/hedvig/productPricing/campaign/service/RedeemCampaignService.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  fun redeemHedvigForever(
```
Comment (meletios) [inline src/main/java/com/hedvig/productPricing/campaign/service/RedeemCampaignService.kt:6]:
```suggestion
  fun enableHedvigForever(
```

Comment (meletios) [reply src/main/java/com/hedvig/productPricing/campaign/service/RedeemCampaignService.kt:6]:
```suggestion
  fun activateHedvigForever(
```

File: src/main/java/com/hedvig/productPricing/campaign/service/RedeemCampaignService.kt (hunk 1 above)
Comment (meletios) [reply src/main/java/com/hedvig/productPricing/campaign/service/RedeemCampaignService.kt:5]: Or it can be in the domainService.

Comment (meletios) [reply src/main/java/com/hedvig/productPricing/campaign/service/RedeemCampaignService.kt:5]:
```suggestion
interface CustomCampaignsService {
```

File: src/main/java/com/hedvig/productPricing/campaign/config/HedvigForeverCampaignConfigurations.kt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.hedvig.productPricing.campaign.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+@ConfigurationProperties(prefix = "hedvig.forever.campaign")
+class HedvigForeverCampaignConfigurations {
+  lateinit var marketMap: Map<String, String>
```
Comment (meletios) [inline src/main/java/com/hedvig/productPricing/campaign/config/HedvigForeverCampaignConfigurations.kt:9]:
```suggestion
  lateinit var customCampaignsMap: Map<String, String>
```

Comment (meletios) [reply src/main/java/com/hedvig/productPricing/campaign/config/HedvigForeverCampaignConfigurations.kt:9]: just a suggestion feel free to ignore it

File: src/test/java/com/hedvig/productPricing/campaign/service/RedeemCampaignServiceImplTest.kt [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    val failedToNotify = classUnderTest.redeemHedvigForever(Market.NORWAY, listOf("7"))
+
+    verify(exactly = 1) { domainService.redeemCampaign(any(), any()) }
+    verify(exactly = 1) { notificationService.sendReferralsEnabled(any()) }
+
+    assertThat(failedToNotify).isEmpty()
+  }
+
+  @Test
+  fun `redeem norwegian hedvig forever campaign tree members`() {
```
Comment (AlexSJ1990) [inline src/test/java/com/hedvig/productPricing/campaign/service/RedeemCampaignServiceImplTest.kt:60]: small typo 'three'

**Pull request merged**